sopel/cli/config.py

```diff
@@ -131,7 +137,12 @@ def handle_init(options):
         return 1
 
     print('Starting Sopel config wizard for: %s' % config_filename)
-    utils.wizard(config_name)
+    try:
+        utils.wizard(config_name)
+    except KeyboardInterrupt:
+        tools.stderr('\nOperation cancelled, no file has been created.')
```

Comment:
```suggestion
        tools.stderr('\nOperation cancelled; no file has been created.')
```

Comma splice bad. x)